apps/docs/components/Aside/Aside.tsx

```diff
@@ -15,22 +17,28 @@ interface AsideProps {
 }
 
 export const Aside: React.FC<AsideProps> = ({ title, links }) => {
-    return (
-        <aside className="hd-aside">
-            <ul className="hd-aside__list">
-                <li className="hd-aside__item hd-aside-section">
-                    <span className="hd-aside__title">{title}</span>
-                    <ul className="hd-aside__nested-list">
-                        {links.map(link => (
-                            <li className="hd-aside__item">
-                                <Link href={link.url} key={link.title} className="hd-aside__link">
-                                    {link.title}
-                                </Link>
-                            </li>
-                        ))}
-                    </ul>
-                </li>
-            </ul>
-        </aside>
-    );
+    const pathname = usePathname();
+
+    if (links.length) {
+        return (
+            <aside className="hd-aside">
+                <ul className="hd-aside__list">
+                    <li className="hd-aside__item hd-aside-section">
+                        <span className="hd-aside__title">{title}</span>
+                        <ul className="hd-aside__nested-list">
+                            {links.map(link => (
+                                <li className="hd-aside__item" key={link.id}>
+                                    <Link href={`${pathname}${link.url}`} className="hd-aside__link">
```

Comment:
There is sadly an issue open regarding links and # and the solution provided for now was to construct the path that way otherwise it goes back to the root page.